build.sbt

```diff
@@ -33,7 +33,7 @@ headerLicense := Some(
   )
 )
 name := "DeltaFlow"
-version := "1.0.1"
+version := "1.0.2"
```

Comment:
Shouldn't version bumping be separated from PR delivering changes? The way it is in [requests](https://github.com/psf/requests/pull/6182/commits/82e63a7c60b707d4d31a8a88416738fd66098c97) via separate PR or in [wvlet/airframe](https://github.com/wvlet/airframe/commit/25774b1fb69727c962d22fce93be514b005118fe) via tags. I don't know which way is better though.

Reply:
I guess we could use sbt-release?